Gem/Code/Source/Components/UI/UiRestBetweenRoundsComponent.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Serialization/EditContext.h>
+
+#if AZ_TRAIT_CLIENT
```

Comment:
The whole component is only on clients, right?

Could one place it only under .Client section without breaking things? AP would be fine but we might see some warnings on the server if the UI canvas is placed directly on the level, maybe?

Reply:
Fixed! Added client_files.cmake